src/components/search/advancedSearch/advancedSearchHelpers/convertToDropdownInput.js

```diff
@@ -30,22 +30,24 @@ export function uniqueDropdownInput(items) {
  * @param prioritisedFormType
  * @param {T} unprioritisedItems
  * @param unprioritisedFormType
+ * @param {("key"|"name")} overrideValueAs
  * @returns {DropdownInputArray}
  */
 export function convertToDropdownInput({
   prioritisedItems,
   prioritisedFormType,
```

Comment:
Bare så jeg selv forstår det. Hvad gør convertToDropdownInput ? :)

Reply:
Den tager input fra f.eks. api (men for nu tager den fra dummy)